### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by blake41**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class PostsController < ApplicationController
+	def index
+		@posts = Post.all
```
Comment (jmburges) [inline app/controllers/posts_controller.rb]: Indentation

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		@post = Post.find(params[:id])
+	end
+
+	def new
+		@post = Post.new
+	end
+
+	def create
+	  @post = Post.new
+	  @post.title = params[:title]
```
Comment (jmburges) [inline app/controllers/posts_controller.rb]: translate to mass assignemtn + strong params

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	  redirect_to post_path(@post)
+	end
+
+	def edit
+		@post = Post.find(params[:id])
+	end
+
+	def update
+	  @post = Post.find(params[:id])
+	  @post.update(title: params[:title], description: params[:description])
```
Comment (jmburges) [inline app/controllers/posts_controller.rb]: mass assignemnt

File: app/views/posts/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<h3>Post Form</h3>
+
+<%= form_tag post_path(@post), method: "put" do %>
```
Comment (jmburges) [inline app/views/posts/edit.html.erb]: Why aren't ehse partials?

Also why not use formhelpers and the `form_for`

File: app/views/posts/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<%= form_tag posts_path do %>
```
Comment (jmburges) [inline app/views/posts/new.html.erb]: `form_for`?

File: config/application.rb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
+    # config.time_zone = 'Central Time (US & Canada)'
+
+    # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
+    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
+    # config.i18n.default_locale = :de
+
+    # Do not swallow errors in after_commit/after_rollback callbacks.
+    config.active_record.raise_in_transactional_callbacks = true
+    config.action_controller.permit_all_parameters = true
```
Comment (jmburges) [inline config/application.rb]: they know strong params

File: spec/features/post_spec.rb [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    expect(page).to have_css("h1", text: "My Post")
+  end
+
+  it 'to post pages' do
+    visit "/posts/#{@post.id}"
+    expect(page.status_code).to eq(200)
+  end
+
+  it 'shows the description on the show page in a p tag' do
+    visit "/posts/#{@post.id}"
```
Comment (jmburges) [inline spec/features/post_spec.rb]: url helpers

**Comment by jmburges** [comment]
few small issues then merge away

**Comment by blake41** [comment]
changes made

**Pull request merged**